test/e2e/generate/helpers.go

```diff
@@ -81,7 +82,7 @@ func createClusteredResource(t *testing.T, client *e2e.E2EClient, resource resou
 	result, err := client.CreateClusteredResource(resource.gvr, &u)
 	Expect(err).NotTo(HaveOccurred())
 	t.Cleanup(func() {
-		deleteResources(client, expectedResource{resource.gvr, result.GetNamespace(), result.GetName()})
+		deleteResources(client, expected(resource.gvr, result.GetNamespace(), result.GetName()))
```

Comment:
Commented in a previous PR, can we custom failure behaviors when deleting namespaces?

Reply:
We could but I didn't hit any issue yet.